stream.go

```diff
@@ -14,6 +14,12 @@ const (
 	errorCodeStoppingGQUIC protocol.ApplicationErrorCode = 7
 )
 
+// The sender is notified by the stream about various events.
+type sender interface {
```

Comment:
Nit: streamSender would make it more clear what the purpose is. Just `sender` is very generic.